src/Files.App/Views/Pages/Properties.xaml

```diff
@@ -4,172 +4,183 @@
 	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:local="using:Files.App.Views"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	d:DesignHeight="700"
-	d:DesignWidth="400"
+	xmlns:toolkitconvers="using:CommunityToolkit.WinUI.UI.Converters"
 	Background="{ThemeResource App.Theme.BackgroundBrush}"
 	KeyDown="Page_KeyDown"
-	Loaded="Properties_Loaded"
-	Unloaded="Properties_Unloaded"
+	Loaded="Page_Loaded"
 	mc:Ignorable="d">
 
-	<Grid VerticalAlignment="Stretch">
-		<Grid.Resources>
-			<ResourceDictionary>
-				<Thickness x:Key="NavigationViewTopPaneHeight">40</Thickness>
-				<Thickness x:Key="TopNavigationViewTopNavGridMargin">0</Thickness>
-				<SolidColorBrush x:Key="NavigationViewContentGridBorderBrush" Color="Transparent" />
+	<Page.Resources>
+		<ResourceDictionary>
 
-				<!--  Removes NavigationView background  -->
-				<SolidColorBrush x:Key="NavigationViewContentBackground" Color="Transparent" />
+			<ResourceDictionary.MergedDictionaries>
+				<ResourceDictionary Source="ms-appx:///ResourceDictionaries/PropertiesStyles.xaml" />
+			</ResourceDictionary.MergedDictionaries>
 
-				<!--  Removes NavigationView corner radius  -->
-				<CornerRadius x:Key="NavigationViewContentGridCornerRadius">0</CornerRadius>
-			</ResourceDictionary>
-		</Grid.Resources>
+			<toolkitconvers:BoolNegationConverter x:Key="BoolNegationConverter" />
+
+			<Thickness x:Key="NavigationViewTopPaneHeight">40</Thickness>
+			<Thickness x:Key="TopNavigationViewTopNavGridMargin">0</Thickness>
+			<SolidColorBrush x:Key="NavigationViewContentGridBorderBrush" Color="Transparent" />
+			<SolidColorBrush x:Key="NavigationViewContentBackground" Color="Transparent" />
+
+		</ResourceDictionary>
+	</Page.Resources>
+
+	<Grid>
 		<Grid.RowDefinitions>
 			<RowDefinition Height="Auto" />
 			<RowDefinition Height="*" />
-			<RowDefinition Height="Auto" />
 		</Grid.RowDefinitions>
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition Width="Auto" />
+			<ColumnDefinition Width="*" />
+		</Grid.ColumnDefinitions>
 
+		<!--  Titlebar region  -->
 		<Border
 			x:Name="TitlebarArea"
-			Grid.Row="0"
-			HorizontalAlignment="Stretch"
+			Grid.ColumnSpan="2"
+			Height="36"
 			Background="Transparent">
+			<StackPanel
+				Padding="12,0"
+				Orientation="Horizontal"
+				Spacing="8">
+
+				<Image
+					Width="18"
+					Height="18"
+					Source="ms-appx:///Assets/AppTiles/Dev/Logo.ico" />
```

Comment:
Can you use one of the logo files here?

Reply:
What do you mean?

Reply:
The ico file is for a specific use case, we should be using one of the logo files that has proper scaling.

Reply:
This will be replaced by its own image like desktop image, folder image, ...?

Reply:
Try using`Square44x44Logo`